tutor/specs/screens/task/index.spec.js

```diff
@@ -0,0 +1,20 @@
+import { Task } from '../../../src/screens/task/index';
+import { Factory } from '../../helpers';
+
+
+describe('Tasks Screen', () => {
+  let props;
+
+  beforeEach(() => {
+    props = {
+      course: Factory.course(),
+    };
+  });
+
+  it('renders', () => {
+    mount(<Task {...props} />);
```

Comment:
what is this testing? should it be `expect(() => mount(<Task {...props} />)).not.toThrow()`?